src/Symfony/Component/Console/Tests/Helper/QuestionHelperTest.php

```diff
@@ -767,9 +768,7 @@ protected function createInputInterfaceMock($interactive = true)
 
     private function hasSttyAvailable()
     {
-        exec('stty 2>&1', $output, $exitcode);
```

Comment:
why not removing the private method entirely ?

Reply:
No particular reason - removed.